app/assets/javascripts/listings/templates/listing/_legal.html.slim

```diff
@@ -1,31 +0,0 @@
-accordion-heading
```

Comment:
was this not in use anywhere?

Reply:
@akegan it was converted into a component and the partial was not deleted -- from the refactor.